.github/workflows/refresh_looker_metrics.yml

```diff
@@ -18,8 +18,6 @@
 name: Refresh Looker Performance Metrics
 
 on:
-  schedule:
-    - cron: '10 10 * * 1'
```

Comment:
> This happens because in Looker Cloud, API3 tokens do not create a full OAuth session, which is required for rendering visualizations. One possible workaround I am considering in the short term is manually logging into Looker via a browser to activate a user session, and then running the workflow (via workflow_dispatch) while that session is still active. ([Tested](https://github.com/Amar3tto/beam/actions/runs/13628737980/job/38091607540) this)

This seems fine for now if we don't have an immediate answer to this problem. Is it possible to obtain an oauth token by following https://cloud.google.com/looker/docs/api-auth#authentication_without_an_sdk ?

Reply:
Already doing this, doesn't work for render methods. I think it will be necessary to consult with @damondouglas. 

@damccorm Could you please try to login into Looker instance (_***/looker.app_) via browser, and then run `workflow_dispatch` [here](https://github.com/Apache/beam/actions/workflows/refresh_looker_metrics.yml) to test if it works?

Reply:
Yes, `OAuth session has expired`. 
This is strange, need to investigate